packages/sui-bundler/webpack.config.prod.js

```diff
@@ -39,7 +39,7 @@ const webpackConfig = {
   name: 'client',
   mode: 'production',
   target,
-  context: path.resolve(CWD, 'src'),
+  context: path.resolve(process.cwd(), 'src'),
```

Comment:
Here you are changing the same value I guess

Reply:
Oh your are right!